api/base_controller.go

```diff
@@ -83,6 +84,13 @@ func (c *BaseController) Routes() []web.Route {
 			},
 			Handler: c.GetSingleObject,
 		},
+		{
+			Endpoint: web.Endpoint{
+				Method: http.MethodGet,
+				Path:   fmt.Sprintf("%s/{%s}%s/{%s}", c.resourceBaseURL, PathParamResourceID, web.OperationsURL, PathParamID),
```

Comment:
It is a bit confusing, because `PathParamResourceID` and `PathParamID` are with changed positions. 
My proposal is to put first `PathParamID` so it is the same as on the other endpoints. 
The other way is to change all other endpoints to use `PathParamResourceID`